### DISCUSSION TITLE
Invalid copilot token: missing token: 403 seems to be back

### DISCUSSION THREAD

**Discussion by UnclAlDeveloper** [community/community#55487] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

This seems to be a repeat of a problem that was flagged here about 6 months ago, and apparently fixed.  I am using copilot with a few different IDEs, Visual Studio and PyCharm, and it is working with them without problem.  I don't use VS Code very often, but I've noticed now that whenever I open it I get the following error/warning:

[INFO] [auth] ... Invalid copilot token: missing token: 403
[Error] [default] ... GitHub Copilot could not connect to server. Extension activation failed: "Thank you for using GitHub Copilot. Your subscription has ended.

I have tried reinstalling copilot etc. but that doesn't resolve the problem.  Possibly I should try and sign out of copilot and re-authenticate, but I can't find any way in VS Code to do this.

**Comment by davecheney** [comment]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by UnclAlDeveloper** [reply]
I'm not 100% sure where you mean to sign out of, I don't have the same graphics for signing out as you have shown, but assume you mean GitHub Desktop.  I signed out of that, and Copilot did ask me to sign back in when VS Code was restarted.  However, I still received the same messages saying that my subscription has ended, along with the invalid/missing token errors.

**Reply by davecheney** [reply]
This screenshot is from VSCode.

**Reply by UnclAlDeveloper** [reply]
I'm still unable to find where your screenshot came from, I'm using VS Code on WIndows (latest version).  It's not under Preferences->Settings, GitHub/Copilot Extensions setup, or anywhere else I could think of.  Might be different versions of VS Code.

**Reply by UnclAlDeveloper** [reply, marked as answer]
Anyway, I seem to have found a resolution, I downloaded the Github-Copilot-Nightly version in place of the standard version, and it now seems to be working.

**Comment by rishi-nuvepro** [comment]
I am facing issue with one of my team member's Copilot user accounts. In VS Code, we are getting the following error, however the same account is working in Visual Studio. I have tried multiple options - sign out, sign in, authorizing the VS Code, uninstalling and installing the Copilot extension, then I have completely removed the Copilot extension and installed the Copilot Nightly version. But nothing worked and I am unable to use this Copilot account - copilot9-nuvepro. 

**[INFO] [auth] [2023-06-14T10:59:35.812Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-06-14T10:59:35.814Z] GitHub Copilot could not connect to server. Extension activation failed: "Contact Support. You are currently logged in as copilot9-nuvepro."**
 
![image](https://github.com/community/community/assets/116337316/06d50f0a-f733-4ec3-81d1-a91c4a0a156f)

I even tried VS Code in a different system to see it helps. There as well I logged in as copilot9-nuvepro user after signing out from the earlier system. Looks to be a problem with this specific GitHub user account that is enabled for Copilot access, and that too not working in VS Code but working in Visual Studio. When I tried Copilot in Visual Studio, the same GitHub login and in the same system, it worked perfect!!

![image](https://github.com/community/community/assets/116337316/13dd4200-0d01-4476-bb6e-3db860954d17)

Any help would be appreciated!!

**Reply by davecheney** [reply]
@rishi-nuvepro please contact support https://support.GitHub.com/contact
